src/Tools/dotnet-counters/KnownData.cs

```diff
@@ -18,20 +18,21 @@ internal static class KnownData
 
         private static IEnumerable<CounterProvider> CreateKnownProviders()
         {
-            yield return new CounterProvider(
-                "System.Runtime", // Name
-                "A default set of performance counters provided by the .NET runtime.", // Description
-                "0xffffffff", // Keywords
-                "5", // Level 
-                new[] { // Counters
-                    new CounterProfile{ Name="cpu-usage", Description="Amount of time the process has utilized the CPU (ms)", DisplayName="CPU Usage (%)" },
-                    new CounterProfile{ Name="working-set", Description="Amount of working set used by the process (MB)", DisplayName="Working Set (MB)" },
-                    new CounterProfile{ Name="gc-heap-size", Description="Total heap size reported by the GC (MB)", DisplayName="GC Heap Size (MB)" },
-                    new CounterProfile{ Name="gen-0-gc-count", Description="Number of Gen 0 GCs / sec", DisplayName="Gen 0 GC / sec" },
-                    new CounterProfile{ Name="gen-1-gc-count", Description="Number of Gen 1 GCs / sec", DisplayName="Gen 1 GC / sec" },
-                    new CounterProfile{ Name="gen-2-gc-count", Description="Number of Gen 2 GCs / sec", DisplayName="Gen 2 GC / sec" },
-                    new CounterProfile{ Name="exception-count", Description="Number of Exceptions / sec", DisplayName="Exceptions / sec" },
-                });
+            return Enumerable.Empty<CounterProvider>();
```

Comment:
Is there any reason you are removing this?

Reply:
No no it's only a draft PR, I did it to for test simulation, I'll revert on "official" PR